PluginControlInterfaceGeneric/CMakeLists.txt

```diff
@@ -17,9 +19,8 @@ OPTION (NSM_IFACE_PRESENT
 
 SET(PLUGIN_LIB_NAME PluginControlInterfaceGeneric)
 
-SET(INCLUDE_FOLDER "include")
-
-SET(CONF_ROOT "${CMAKE_INSTALL_PREFIX}/etc/controllerconf")
+set(CONTROLLER_CONFIG_DIR "${CMAKE_INSTALL_FULL_SYSCONFDIR}/${LIB_INSTALL_SUFFIX}/control"
```

Comment:
Hello Isogai-san,
The path will expand, following the comment, to /usr/local/etc/audiomanager/control. But in the end there will be one Controller attached to the AudioManager, which in the end has to be single instance. Just asking, is it necessary to store the configuration in a /usr/local --> "per user" folder?